src/ngrx/7-create-api-effects/create-api-effects.md

```diff
@@ -21,11 +21,13 @@
 
 3. Dispatch `LoginActions.logoutFailure` when API request fails
 
-## Problem 1: TODO
+## Problem 1: Create login$ Effect to Handle API Requests
```

Comment:
Do we want to use backticks on variable names in headers? Would doing so require a lot of changes?

Reply:
naw this makes sense. Members, variables, code stuff should use backticks